substrate/frame/revive/src/lib.rs

```diff
@@ -406,6 +406,9 @@ pub mod pallet {
 			/// Data supplied by the contract. Metadata generated during contract compilation
 			/// is needed to decode it.
 			data: Vec<u8>,
+			/// A list of topics used to index the event.
+			/// Number of topics is capped by [`limits::NUM_EVENT_TOPICS`].
```

Comment:
Should we just hard code the max to 4, as this is what you can have in EVM 🤔

Reply:
Using a constant is already hard coding. How would you hard code it harder? Using a `BoundedVec`?

Reply:
ah nvm I though it was configurable for some reason